Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml

```diff
@@ -0,0 +1,69 @@
+- type: entity
+  name: space shrimp
+  parent: [ SimpleMobBase, FlyingMobBase, MobCombat ]
+  id: MobSpaceShrimp
+  description: Shrimply cursed...
+  suffix: METEM MACHINE ONLY #for now, only to be used as a very rare result for the metempsychosis machine. If fishing ever gets added, this suffix can be removed.
+  components:
+  - type: CombatMode
```

Comment:
Doesn't the parented `MobCombat` have this?

Reply:
I don't know, I was high when I made this